Makefile

```diff
@@ -121,33 +121,7 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	mkdir -p $(IB_BUILD_DIR)
 	$(eval TOOLCHAIN_PATH := $(shell printf "%s:" $(OPENWRT_DIR)/staging_dir/toolchain-*/bin))
 	$(eval IB_FILE := $(shell ls -tr $(OPENWRT_DIR)/bin/$(MAINTARGET)/OpenWrt-ImageBuilder-*.tar.bz2 | tail -n1))
-	cd $(IB_BUILD_DIR); tar xf $(IB_FILE)
-	# shorten dir name to prevent too long paths
-	mv $(IB_BUILD_DIR)/$(shell basename $(IB_FILE) .tar.bz2) $(IB_BUILD_DIR)/imgbldr
-	export PATH=$(PATH):$(TOOLCHAIN_PATH); \
-	PACKAGES_PATH="$(FW_DIR)/packages"; \
-	for PROFILE_ITER in $(PROFILES); do \
-	  for PACKAGES_FILE in $(PACKAGES_LIST_DEFAULT); do \
-	    PROFILE=$$PROFILE_ITER \
-	    CUSTOM_POSTINST_PARAM=""; \
-	    if [[ $$PROFILE =~ ":" ]]; then \
-	      SUFFIX="$$(echo $$PROFILE | cut -d':' -f 2)"; \
-	      PACKAGES_SUFFIXED="$${PACKAGES_FILE}_$${SUFFIX}"; \
-	      if [[ -f "$$PACKAGES_PATH/$$PACKAGES_SUFFIXED.txt" ]]; then \
-	        PACKAGES_FILE="$$PACKAGES_SUFFIXED"; \
-	        PROFILE=$$(echo $$PROFILE | cut -d':' -f 1); \
-	      fi; \
-	    fi; \
-	    if [[ -f "$$PACKAGES_PATH/$$PACKAGES_FILE.sh" ]]; then \
-	      CUSTOM_POSTINST_PARAM="CUSTOM_POSTINST_SCRIPT=$$PACKAGES_PATH/$$PACKAGES_FILE.sh"; \
-	    fi; \
-	    PACKAGES_FILE_ABS="$$PACKAGES_PATH/$$PACKAGES_FILE.txt"; \
-	    PACKAGES_LIST=$$(grep -v '^\#' $$PACKAGES_FILE_ABS | tr -t '\n' ' '); \
-	    $(UMASK);\
-	    echo -e "\n *** Building Kathleen image file for profile \"$${PROFILE}\" with packages list \"$${PACKAGES_FILE}\".\n"; \
-	    $(MAKE) -C $(IB_BUILD_DIR)/imgbldr image PROFILE="$$PROFILE" PACKAGES="$$PACKAGES_LIST" BIN_DIR="$(IB_BUILD_DIR)/imgbldr/bin/$$PACKAGES_FILE" $$CUSTOM_POSTINST_PARAM || exit 1; \
-	  done; \
-	done
+	#mv $(IB_BUILD_DIR)/$(shell basename $(IB_FILE) .tar.bz2) $(IB_BUILD_DIR)/imgbldr
```

Comment:
Commenting out this but at the same time removing the explaining comment isn't very good style.

Also, it's not like shortening the path was added because we were just bored; it was necessary.
Have you tried building in, e.g., /home/user/project/freifunk/firmware? Previously, the build failed then (also on the buildbot, for some targets).